odl/trafos/wavelet.py

```diff
@@ -136,123 +136,26 @@ def coeff_size_list(shape, nscales, wbasis, mode):
     return size_list
 
 
-def pywt_coeff_to_array2d(coeff, size_list):
+def pywt_coeff_to_array(coeff, size_list):
     """Convert a `pywt
     <http://www.pybytes.com/pywavelets/>`_ coefficient into a flat array.
 
-    Related to 2D multilevel discrete wavelet transform.
+    Related to 2D and 3D multilevel discrete wavelet transform.
 
     Parameters
     ----------
+    1D:
```

Comment:
You need to structure this a bit differently due to the way the doc parser reads the text. It will interpret `1D` as a parameter, so you cannot insert extra headers between the parameters. 

Instead make sections in the text under `coeff`.